ac1/_sources/regression_and_line_of_best_fit/interpreting_slope.rst

```diff
@@ -8,78 +8,100 @@
 Interpreting Slope
 ==================
 
-When interpreting the relationship between two variables, one of the key
-concepts is interpreting slope. While the correlation coefficient tells you how
-strong the relationship between two variables is, the slope tells you the
-magnitude of that relationship. In other words, the correlation helps you answer
-the question “If variable A changes, will variable B change?”, while the slope
-helps you answer the question “If variable A changes, *how much* will variable B
-change?”.
+Now that you have reviewed the section on equation of a line, it's important to
+see how this information relates to data analysis. As mentioned in the previous
+section, the slope gives the steepness of a line, but what does this mean?
 
-Suppose two schools are pretty similar in most regards, except that College X
-has an average SAT math score that is 100 points higher than University Y.
-Knowing the slope of the line of best fit in this case is useful to predict how
-SAT score impacts other variables.
+You can start to answer this question by first understanding what slope is. The
+`slope of a straight line <https://en.wikipedia.org/wiki/Slope>`__ tells you
+how much the y value changes when the x value is increased by 1. This is also
+written as the ratio
 
-For example, below is a scatter plot with median SAT Math score as the
-explanatory variable, and median earnings after graduation as the explained
-variable.
 
+.. math::
+   \frac{rise}{run} = \frac{\text{change in y}}{\text{change in x}}
 
-.. image:: figures/SAT_Math_and_Earnings.png
 
+You've already learned about the correlation coefficient, *r*, which tells you
+how strong the relationship between two variables is. Slope tells you something
+different. The slope tells you the **magnitude** of that relationship. In other
+words, the correlation helps you answer the question “If variable A changes,
+*will* variable B change?”, while the slope helps you answer the question “If
```

Comment:
Slightly change the wording

"If variable A changes, how confident are you that variable B will change?" or something like that